src/openforms/js/components/admin/RJSFWrapper.js

```diff
@@ -71,14 +77,43 @@ const CustomCheckboxWidget = ({
     value,
     disabled,
     readonly,
+    label,
     autofocus,
     onChange,
+    ...props
 }) => {
+
+    // if it's nullable, defer to a dropdown
+    if (isSelect(schema)) {
+        const enumOptions = optionsList(schema);
+        for (let option of enumOptions) {
+            option.value = JSON.stringify(option.value);
+        }
+        const stringValue = typeof value != 'string' ? JSON.stringify(value) : value;
+        const _onChange = (value) => {
+            onChange(JSON.parse(value));
+        };
+        return (
+            <Widgets.SelectWidget
+                {...props}
+                options={{...props.options, enumOptions}}
+                id={id}
+                schema={{default: 'null', ...schema}}
+                value={stringValue}
+                disabled={disabled}
+                readonly={readonly}
+                label={label}
+                autofocus={autofocus}
+                onChange={_onChange}
+            />
```

Comment:
I was wondering why you added this code specific to the SelectWidget inside of the Checkbox widget instead of customising the the rjsf select widget? https://github.com/rjsf-team/react-jsonschema-form/blob/master/packages/core/src/components/widgets/SelectWidget.js#L49